src/aiq/profiler/inference_optimization/bottleneck_analysis/nested_stack_analysis.py

```diff
@@ -303,6 +303,8 @@ def save_gantt_chart(all_nodes: list[CallNode], output_path: str) -> None:
 
     # Sort calls by start_time
     sorted_nodes = sorted(all_nodes, key=lambda x: x.start_time)
+    min_start = min(node.start_time for node in sorted_nodes)
```

Comment:
Since `sorted_notes` is already sorted by `start_time` I believe the `min` isn't needed here and could just be:
```suggestion
    min_start = sorted_nodes[0]
```

Reply:
Good point. Fixing